upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -111,6 +111,39 @@ There you can find links to upgrade notes for other versions too.
     -           ->setExpectedStatusCode(404);
     +           ->setExpectedStatusCode(302);
     ```
+- contact form has been moved to separate page. You can find the whole new setting in administration (`/admin/contact-form/`),
+  where you can edit main text for contact form. ([#1522](https://github.com/shopsys/shopsys/pull/1522))
+  
+  There are few steps need to be done, to make contact form work on FE
+    - in `ContactFormController` change previous `indexAction` with [the new one](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Controller/Front/ContactFormController.php). Please pay attention if you have some modification in previous implementation.
+    - in `ContactFormController` remove action `sendAction()`, it is not needed anymore
+    - remove `src/Resources/scripts/frontend/contactForm.js`, it is not needed anymore
+    - new localized route `front_contact` has been introduced with slug `contact`. This slug can be already in use in your project, because
+      previous SSFW versions have an article called `Contact` which was used as the contact page. Please move the article's content to the new contact page and completely remove the article.
+      To remove the article, please create [new migration](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Migrations/Version20191121171000.php) in you project which removes the article and its slug.
+      
+      !!! hint
+          If you don't want to remove the article, you will need to change path for the new route. See next step
+    
+    - add new localized route in all your localized routing `yml` files with translated `path` option
+        ```diff 
+        +    front_contact:
+        +    path: /contact/
+        +    defaults: { _controller: ShopsysShopBundle:Front\ContactForm:index }
+        ```
+    - add new template [`templates/Front/Content/ContactForm/index.html.twig`](https://github.com/shopsys/shopsys/blob/9.0/project-base/templates/Front/Content/ContactForm/index.html.twig)
+    - add link to new contact page somewhere in templates (e.g in `footer.html.twig`)
+        ```diff
+            <div class="footer__bottom__articles">
+               {{ getShopInfoPhoneNumber() }}
+               {{ getShopInfoEmail() }}
+               {{ render(controller('App\\Controller\\Front\\ArticleController:footerAction')) }}
+        +      <a class="menu__item__link" href="{{ url('front_contact') }}">{{ 'Contact'|trans }}</a>
+            </div>
+        ```
+    - it is recommended to create database migration to remove `contact` slug from `friendly urls` database table, to make new url work. 
+    Also if you do not need article with contact, you can remove it too        
```

Comment:
This is not necessary anymore as it's handled a few lines above.

Reply:
fixed